### PR TITLE
Update WinRT contracts per specs\WinRT\WinRTAPIContracts.md

### PR DESCRIPTION
**Pull request by DrusTheAxe**
Update WinRT contracts per specs\WinRT\WinRTAPIContracts.md (some contract names didn't match or were absent)

### CODE REVIEW

**Review by dhoehna — APPROVED** [review]
Environment Manager looks good.

**Review by loneursid — APPROVED** [review]
LGTM for the AppNotifications contract

File: dev/MRTCore/mrt/Microsoft.Windows.ApplicationModel.Resources/src/Microsoft.Windows.ApplicationModel.Resources.idl [hunk 1]
```diff
@@ -4,9 +4,9 @@
 namespace Microsoft.Windows.ApplicationModel.Resources
 {
     [contractversion(1.0)]
-    apicontract MrtContract{};
+    apicontract MrtCoreContract{};
 
-    [contract(MrtContract, 1.0)]
+    [contract(MrtCoreContract, 1.0)]
```
Comment (huichen123) [inline dev/MRTCore/mrt/Microsoft.Windows.ApplicationModel.Resources/src/Microsoft.Windows.ApplicationModel.Resources.idl:9]: can you change 1.0 to 1?

**Review by huzaifa-d — APPROVED** [review]

**Comment by huzaifa-d** [comment]
LGTM for PowerNotifications

File: dev/MRTCore/mrt/Microsoft.Windows.ApplicationModel.Resources/src/Microsoft.Windows.ApplicationModel.Resources.idl (hunk 1 above)
Comment (huichen123) [reply dev/MRTCore/mrt/Microsoft.Windows.ApplicationModel.Resources/src/Microsoft.Windows.ApplicationModel.Resources.idl:9]: is this resolved?

Comment (DrusTheAxe) [reply dev/MRTCore/mrt/Microsoft.Windows.ApplicationModel.Resources/src/Microsoft.Windows.ApplicationModel.Resources.idl:9]: Doh! It was, but I forgot to push the commit :-(

Fixed by https://github.com/microsoft/WindowsAppSDK/pull/2384

**Pull request merged**